include/Util/Input.hpp

```diff
@@ -50,7 +52,7 @@ class Input {
      *
      * @see Util::Input::SetCursorPosition()
      */
-    static glm::vec2 GetCursorPosition();
+    static Util::PTSDPosition GetCursorPosition();
```

Comment:
I'm considering making this function return `SDLPosition` because `SDLPosition` can be safely & implicitly converted to `PTSDPosition` (actually lossless here), making the return value more flexible to use.

Reply:
IMO, since you already defined the PTSDPosition, it may be better and easy for the users to use PTSDPosition instead of SDLPosition, rather decide which Position class should use.

Reply:
Based on @HEKEPOIU's suggestion, I believe that some developers may prefer to use `SDLPosition`, and I prefer to provide more options for developers.

Reply:
And `PTSDPosition` may be important for the future camera and alignment features, which are the parts I'm not sure about yet.